docs/getting-started.md

```diff
@@ -239,68 +265,50 @@ import Kilt, {
-  nonce: '0xcf91336fe836f9ab9d99c43025b06e411d57b636d1a17fd0',
-  hash:
-   '0x45a03837bf67318a0e8b97e057cf347cf6dcfc280cc89fbf601a23bd7fe1b3e2',
-  signature:
-   '0xefdfbbac968ec805c22bfe97219268d470cfb74e27ae13ef29c399a74ffae68d55a0a474b6c5ecea37e4716c0ec81dc83761ee99b13a8c984ea4c76101bab204' }
+ Message {
+  body: {
+    content: { requestForAttestation: [RequestForAttestation] },
+    type: 'request-attestation-for-claim'
+  },
+  createdAt: 1595252779597,
+  receiverAddress: '5HTEzvVT5bQxJTYPiDhRUw4GHarQVs66sFQEpQDUNT6MyoJr',
+  senderAddress: '5EoUcwSZm4KgtLCN8SBYoXXv5p3b9gKS58tiRbtJidrhMp3b',
+  senderBoxPublicKey: '0x9e5869608be42588504aaa4c55cb24c17ad1af38c37f821695a93e74049aa112',
+  message: '0x3299211b4ab4b34fc5a753223349bde368a3ab3a6cc4fb516b90e4516a1958372924500959e8e78c3b7a073fc6474156d9d926651c006c2603e005a07a20b26a9eb811386f703080868237e1b27e03a3f5d0240a3d11fa437089579f587ccd62b3183f3a5b5552ac9a0979cad7bff4aa317f7923d939e6983070ed1a86744d78eac2705cbc91767f21d0e62c005a5e3e98b13ae0d4f25a1580a29aa44f17de0ac708afae6df8eceb1a9921e2e09c7cadd9036b5c47c634af1ddf44f39bd7b6f143d6990bfd0d16fad24316d3d4e9e66910976ddc3fc2ab99bfa5bf83965767173a7e9553d0fa48145cb925b2c61fd4d2f3a3d49c7553ece7f76cbc1f62d5e8746af442720cd00261abf611cd42462536bdc0bfdd27f4e9a40c7a6994cd90105f740d34537c7119fb0a19d1e7ecd7d9054a0868af83bc03a971841b74722e16b48055819c3ab23bb61b75d67a3dd8c9db383ed8c1f21e1db20f66ce96ca74eab571cb1021ef0171746f2e8642efbdf317f29aa4e3dde41dcb9c40ba7e56bd51a4d7a0928a10a8f272ab2651d40acdc25b6513cf67e17f4cbe554224242805df6026e357feaabad80573b3d68450f2c3b7019b52010369eb5a943c63cea8e933ab9dfe8b441fbc2c5c84da11eceb93bc756ffcf7dfed4f3fd4de0b4358b9a551592172832ffe1ebcff33e82c2a12861b91060c754ae70df7bf11463db53d3256335e89aca78934003e65b2e67647fdaedd6335a98d403422442840d071efc48a38708870ea2a9172587c70c4a9c086c25667fa9a0be04cb3f92cacbfbd53811af23fd3465a5e9a85906a628377b5f0620404782056373134799e7a8afb6f505e08c17931f987a1b9686977c4874be827d92f2ec915ba072de3092ddf8bc6140b70e24a0e3662bed23c46a1fbea9d1ed03ed69812f607e153a8ef8c2bc298498ab4c5163c1bd729d56ed72a3d6ffdeced9e5f1ed9242223f421d398386188b1bfdbb2d97495dcd11b327d441ebe4cb3a855ef37bbd8c608c646a215014025203e7b249d4a579b7a65bfa473e300322f24c21f6adfe7298b564e23e35e02f02a97530831e5533368ac55eae19165d768e353ad3063fd06187b4557f5f1e878088cb481e60cc4ccc2723cefcc10ae5c888170911d195f43e8539a4c01b04ca05235d19793fb6c29886e0c3472a47d2365c4a78b2dbddc6e7f446c26e6ae5f2801f8552f8cd76817943e067421341b27de63bc82e69c579fa789df35f2b33266e3847865e360b494ef4053b4dad8975d0cc2fef28409c369ea74c13627235c150e5c9d3a105c0059e9a6ed03be59ba9e2cf9a4dd0e13e081d8ed52dc7aeff4edf59290d373800e379dacbf2b7f82e13cf9a2542e5bf4aa41f11b4173cc35cbcd4cf8a3c74dfb39e53e67df32ae8b1f2177b227e2194a5b9a7d1ee65bf606678338bff6e87c628a5b65539a62397ebc5f7e39f62638535b8d3d6a0c27a4d1a8e97f0577d61c036351333d2cacaee54533b0ca9169d3cb309dad6926b677f654bc0febbc2da3cbc36be6dc5c77b99270189d51259280683b9782180eac1b735fb6b365ad4f9980c890f6e3f4d5b7120b9d2dd845',
```

Comment:
maybe omit the whole message like that: `message: '0xFEED...CAFE`

Reply:
Fixed in e1f2081